prov/util/src/util_shm.c

```diff
@@ -280,13 +281,11 @@ int smr_create(const struct fi_provider *prov, struct smr_map *map,
 	(*smr)->peer_data_offset = peer_data_offset;
 	(*smr)->name_offset = name_offset;
 	(*smr)->sock_name_offset = sock_name_offset;
-	(*smr)->cmd_cnt = rx_size;
```

Comment:
Let's add an assert on ep close to make sure all our commands are accounted for at the end. Just to catch any cmd_cnt++/-- issues.